docpage/docs/Users.md

```diff
@@ -90,7 +91,7 @@ components:
 
 #### Alterations
 
-The subsection `alterations`, inside Sources, change the data format. These alterations are applied as a descendent definition. In this example, the input data is merged into one message. Then, the merge message is encoded.
+The subsection [Alterations](/docs/Alterations), is located inside [Sources](/docs/Sources), and it changes the data format. These alterations are applied as a descendent definition. In this example, the input data is merged into one message. Then, the merge message is encoded in base64 format.
```

Comment:
[Alterations](/docs/Alterations), located inside [Sources](/docs/Sources), are employed to modify the format of data. The alterations are applied in the specified order. In the following example, the input data is merged into one message. Then, the merged message is encoded in base64 format.